### PR TITLE
Add support to build Node.js CS3API bindings and push them to GitHub

### PR DESCRIPTION
**Pull request by SamuAlfageme**
Needs more testing and the companion PR in cs3org/node-cs3apis to enable releasing the generated bindings to npm

### CODE REVIEW

**Comment by SamuAlfageme** [comment]
cc/ @piotrWichlinskiSoftwaremind

File: build.go [hunk 1]
```diff
@@ -438,6 +479,10 @@ func pushJS() {
 	push("build/js-cs3apis")
 }
 
+func pushNode() {
+	push("build/node-cs3apis")
+}
+
 func main() {
 	if *_buildProto {
 		fmt.Println("Compiling and liniting protobufs ...")
```
Comment (ishank011) [inline build.go]: @SamuAlfageme can you fix the typo? 😁

**Comment by ishank011** [comment]
Also can you rebase and add the command to remove existing defs? I added that recently

**Comment by SamuAlfageme** [comment]
@ishank011 totally missed it! You got it.

**Review by ishank011 — APPROVED** [review]

**Pull request merged**